app/scripts/HiGlassComponent.js

```diff
@@ -831,7 +835,37 @@ class HiGlassComponent extends React.Component {
     }
   }
 
+  toggleTypedEditable() {
+    this.setState({
+      typedEditable: !this.isEditable()
+    });
+  }
+
+  /** Handle typed commands (e.g. e-d-i-t) */
+  typedTextHandler(event) {
+    this.typedText = this.typedText.concat(event.key);
```

Comment:
Just out of curiosity, why not simply `this.typedText.push(event.key)`?

Reply:
No good reason.